### PR TITLE
Add missing word to text of the alert dialog

### LINKED ISSUE #79380: Missing word in alert popup
### Godot version

v4.1.stable.official

### System information

Godot v4.1.stable - Windows 10.0.19045 - Vulkan (Forward+) - dedicated AMD Radeon (TM) RX 480 (Advanced Micro Devices, Inc.; 31.0.14057.5006) - Intel(R) Core(TM) i7-6700 CPU @ 3.40GHz (8 Threads)

### Issue description

There is a missing word in an alert popup. Will include a screenshot.

The alert should read, "Missing file '(filename' or one of its dependencies".

Instead it reads, "Missing file '(filename' or one its dependencies".

(it's missing the word "of").

NOTE: THE IMPORT ERROR IS NOT THE ISSUE, ITS THE WORDING IN THE ALERT POPUP.

This report should be of LOW priority.

(p.s. If I'm missing any info you'll have to forgive me; this is my first issue/bug report.

image

### Steps to reproduce

Using your file manager, copy a scene with a script attached (just the .tscn and .gd files) into a different project's root res:// folder.

Then try opening the pasted scene and you will get the alert.

### Minimal reproduction project

N/A requires two projects.

### PR DESCRIPTION
**Pull request by WiseNoodle**
I hope I followed all the guidelines correctly.

* *Bugsquad edit, fixes: #79380*

### CODE REVIEW

File: editor/editor_node.cpp [hunk 1]
```diff
@@ -1411,7 +1411,7 @@ void EditorNode::_dialog_display_load_error(String p_file, Error p_error) {
 				show_accept(vformat(TTR("Scene file '%s' appears to be invalid/corrupt."), p_file.get_file()), TTR("OK"));
 			} break;
 			case ERR_FILE_NOT_FOUND: {
-				show_accept(vformat(TTR("Missing file '%s' or one its dependencies."), p_file.get_file()), TTR("OK"));
+				show_accept(vformat(TTR("Missing file '%s' or one of its dependencies."), p_file.get_file()), TTR("OK")); // added the word "of" to this string for clarity
```
Comment (AThousandShips) [inline editor/editor_node.cpp:1414]:
```suggestion
				show_accept(vformat(TTR("Missing file '%s' or one of its dependencies."), p_file.get_file()), TTR("OK"));
```
Comment is unnecessary

**Comment by WiseNoodle** [comment]
Reverting commits is confusing. If I didn't do It correctly close this and i'll redo it

**Comment by AThousandShips** [comment]
You will need to do `git rebase -i master` and then write "fixup" in front of the other commits

**Comment by WiseNoodle** [comment]
Like so? (forgive me I'm still learning git)
image

**Comment by AThousandShips** [comment]
No replace "pick" with "fixup" except for the first one

**Comment by AThousandShips** [comment]
Then you need to do `git push -f`

**Comment by AThousandShips** [comment]
If you're having problems I can try and do it from my end in a bit

**Comment by WiseNoodle** [comment]
I'm trying my best lol, I'm kinda lost rn; It says everything is up to date when i push

**Comment by WiseNoodle** [comment]
Oh boy, i have definitely messed something up
image

**Comment by AThousandShips** [comment]
You wrote "it" instead of "git"

**Comment by WiseNoodle** [comment]
this is painful for you to watch im sure, my apologies.

**Comment by Sauermann** [comment]
Don't worry, git has a bit of a learning curve. But it helps to learn the basics.

**Comment by AThousandShips** [comment]
Not at all, there was a time when I was getting used to it too

**Comment by WiseNoodle** [comment]
(recloning, give me a few)

**Comment by WiseNoodle** [comment]
Okay, it finished downloading. I have a fresh slate. I just need to sync it with what we've done so far right?

**Comment by AThousandShips** [comment]
Yes you just pull this branch from your repository

**Comment by WiseNoodle** [comment]
now jes gotta change the text...

**Comment by WiseNoodle** [comment]
it is XD

**Comment by WiseNoodle** [comment]
What's GitHub doing right now? Is it checking all os versions?? That's so cool

**Comment by AThousandShips** [comment]
It's running a lot of automated tests! Compiling it all and stuff, it's really cool

**Comment by WiseNoodle** [comment]
That's amazing, so is there anything else I need to do?

**Review by AThousandShips — APPROVED** [review]

**Comment by WiseNoodle** [comment]
In hindsight, I should have compiled and run a test of it before comitting it... But it's just a sting of text, it should be fine if the dialog was made to scale with the text.

**Comment by AThousandShips** [comment]
In the future that's a great thing to do, to catch errors and things

**Comment by WiseNoodle** [comment]
Question: Does this change get applied in the next update (e.g. 4.1.1 or 4.2)?

**Comment by YuriSizov** [comment]
> Question: Does this change get applied in the next update (e.g. 4.1.1 or 4.2)?

It will probably be included in both.

FYI, the email that you've used to author your commit is invalid (`wisenoodle21@gmail`, missing `.com`). Which means it will not be counted towards your personal contributions on GitHub. You should be able to fix that using `git commit --amend --author="Your Name <your_correct_email>"`. You should also change your local git settings to make sure this doesn't happen in future.

PS. Please use the "Edit" button on GitHub and avoid multiposting. Each message that you send creates a notification for a lot of people. If you need live assistance, consider joining contributors chat at https://chat.godotengine.org!

**Comment by AThousandShips** [comment]
You deleted the message, please update it to fit the PR title with `git commit --amend`

**Review by YuriSizov — APPROVED** [review]

**Comment by YuriSizov** [comment]
Thanks! And congrats on your first merged Godot PR!

**Comment by AThousandShips** [comment]
Congratulations!

**Comment by WiseNoodle** [comment]
Thank you all so much for all your help, I hope to improve my git skills and continue to contribute in the future!

**Comment by YuriSizov** [comment]
Cherry-picked for 4.1.1.

**Pull request merged**